### PR TITLE
comment&changes

### PR DESCRIPTION
**Pull request by yparks**
comments and delete userfragment

### CODE REVIEW

File: app/src/main/java/edu/mills/cs115/BaseFragment.java [hunk 1]
```diff
@@ -9,15 +9,15 @@
 import android.view.ViewGroup;
 
 /**
- *
+ *BaseFragment needs extends
```
Comment (RobertoAF) [inline app/src/main/java/edu/mills/cs115/BaseFragment.java]: Missing space between * and Base

File: app/src/main/java/edu/mills/cs115/BaseFragment.java [hunk 2]
```diff
@@ -9,15 +9,15 @@
...
  *
  */
 public abstract class BaseFragment extends Fragment {
 
   protected Context mContext;
 
   /**
-   *
+   *Created this class by Callback
```
Comment (RobertoAF) [inline app/src/main/java/edu/mills/cs115/BaseFragment.java:20]: Missing space

File: app/src/main/java/edu/mills/cs115/BaseFragment.java [hunk 3]
```diff
@@ -31,20 +31,21 @@ public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container,
   }
 
   /**
-   *
+   *Abstract class is declared, may not be instantiated, and require subclass to provide implementation
```
Comment (RobertoAF) [inline app/src/main/java/edu/mills/cs115/BaseFragment.java]: Missing space

File: app/src/main/java/edu/mills/cs115/BaseFragment.java [hunk 4]
```diff
@@ -31,20 +31,21 @@ public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container,
...
+   *for the anstract methods.
```
Comment (RobertoAF) [inline app/src/main/java/edu/mills/cs115/BaseFragment.java]: Missing space

File: app/src/main/java/edu/mills/cs115/BaseFragment.java [hunk 5]
```diff
@@ -31,20 +31,21 @@ public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container,
...
    */
   public abstract View initView();
 
   /**
-   *
+   *Created Activity by Callack this method
```
Comment (RobertoAF) [inline app/src/main/java/edu/mills/cs115/BaseFragment.java]: Missing space

File: app/src/main/java/edu/mills/cs115/BaseFragment.java [hunk 6]
```diff
@@ -31,20 +31,21 @@ public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container,
...
    */
   @Override public void onActivityCreated(@Nullable Bundle savedInstanceState) {
     super.onActivityCreated(savedInstanceState);
     initData();
   }
 
   /**
-   *
+   *When subclass needs connect with internt, rewrite this method to connect with internet
```
Comment (RobertoAF) [inline app/src/main/java/edu/mills/cs115/BaseFragment.java]: Missing space

File: app/src/main/java/edu/mills/cs115/DictionaryBean.java [hunk 7]
```diff
@@ -1,5 +1,9 @@
 package edu.mills.cs115;
 
+/**
+ *@author Ying Parks
```
Comment (RobertoAF) [inline app/src/main/java/edu/mills/cs115/DictionaryBean.java]: Missing space

File: app/src/main/java/edu/mills/cs115/HomeFragment.java [hunk 8]
```diff
@@ -77,7 +84,7 @@
         return new NetworkImageHolderView();
       }
     };
-    //
+    //Randome generate 4 memes pictures for banner
```
Comment (RobertoAF) [inline app/src/main/java/edu/mills/cs115/HomeFragment.java:87]: Random

File: app/src/main/java/edu/mills/cs115/HomeFragment.java [hunk 9]
```diff
@@ -107,7 +114,7 @@
       banner.stopTurning();
     }
   }
-
+// Randome generate a word  as word of a day content
```
Comment (RobertoAF) [inline app/src/main/java/edu/mills/cs115/HomeFragment.java]: Random

File: app/src/main/java/edu/mills/cs115/HomeFragment.java [hunk 10]
```diff
@@ -157,7 +164,7 @@ public DictionaryBean queryRandom() {
       Cursor cursor =
           db.rawQuery("SELECT * FROM " + DictionaryDatabaseHelper.DICTIONARY_TABLE, null);
       Log.v("MainActivity.java", "in doTermSearch() " + cursor.getCount());
-      //
+      //Randome generate a word form database
```
Comment (RobertoAF) [inline app/src/main/java/edu/mills/cs115/HomeFragment.java]: Random

**Review by RobertoAF — CHANGES_REQUESTED** [review]

File: app/src/main/java/edu/mills/cs115/HomeFragment.java (hunk 8 above)
Comment (espertus) [reply app/src/main/java/edu/mills/cs115/HomeFragment.java:87]: Actually, "Randomly".

**Pull request closed without merging**